include/quicly.h

```diff
@@ -435,6 +435,18 @@ struct st_quicly_conn_streamgroup_state_t {
          * Total bytes sent, at UDP datagram-level.                                                                                \
          */                                                                                                                        \
         uint64_t sent;                                                                                                             \
+        /**                                                                                                                        \
+         * Total bytes sent but lost, at UDP datagram-level.                                                                       \
+         */                                                                                                                        \
+        uint64_t lost;                                                                                                             \
+        /**                                                                                                                        \
+         * Total STREAM_DATA bytes sent                                                                                            \
```

Comment:
Considering that we do not have a definition for "STREAM_DATA," maybe something like below?
```suggestion
         * Total amount of stream-level payload being sent \
```